discojs/src/aggregator/get.ts

```diff
@@ -33,6 +34,42 @@ export function getAggregator(
   const scheme = options.scheme ?? task.trainingInformation.scheme
   
   switch (task.trainingInformation.aggregationStrategy) {
+    case 'byzantine': {
+        const {
+          clippingRadius = 1.0,
+          maxIterations = 1,
+          beta = 0.9,
+        }:{
+            clippingRadius?: number;
+            maxIterations?: number;
+            beta?: number;
+          } = task.trainingInformation;
```

Comment:
no need to explicit the type, it is already inferred

```suggestion
        } = task.trainingInformation;
```